lib/ember-test-helpers/test-module.js

```diff
@@ -230,6 +230,13 @@ export default Klass.extend({
     this.container = items.container;
     this.registry = items.registry;
 
+
+    var viewRegistry = this.container.lookup('-view-registry:main');
+    if (viewRegistry) {
+      var View = Ember.__loader.require('ember-views/views/view').default;
+      View.views = viewRegistry;
```

Comment:
No, I don't think something like that exists, but the following code should work for us [here](https://github.com/switchfly/ember-test-helpers/blob/704e91f826f1a5ef1dcb50f541f3d5e188676492/lib/ember-test-helpers/test-module-for-component.js#L113):

```javascript
var thingToRegisterWith = this.registry || this.container;
thingToRegisterWith.register('component:-template-under-test', Ember.Component.extend({
  layout: template
});
module.component = this.container.lookup('component:-template-under-test');
```